README.md

```diff
@@ -21,6 +21,7 @@ compiling quantum programs in clever ways.
 Want to know more? 
 - Check out our
 [documentation](https://mitiq.readthedocs.io/en/stable/guide/guide.html).
+- To see what's in store for Mitiq this year, look at the 2024 Roadmap in the [wiki](https://github.com/unitaryfund/mitiq/wiki).
```

Comment:
By referencing a specific year, the main README risks becoming outdated without us noticing when the year ends, giving the impression that it's old or hasn't been updated.